bitmind/synthetic_data_generation/synthetic_data_generator.py

```diff
@@ -250,12 +257,18 @@ def run_t2vis(
                 self.t2vis_model
             )
 
-            torch_dtype =  T2VIS_MODELS[self.t2vis_model_name].get(
-                'from_pretrained_args', {}).get('torch_dtype', torch.bfloat16)
-
-            bt.logging.info("Generating media from prompt")
+            bt.logging.info(f"Generating media from prompt: {truncated_prompt}")
+            bt.logging.info(f"Generation args: {gen_args}")
             start_time = time.time()
-            with torch.autocast(self.device, torch_dtype, cache_enabled=False):
+            if model_config.get('use_autocast', True):
```

Comment:
autocast causes SDXL to output black images, but is required for video models (crashes otherwise)

Reply:
made it conditional, only turned off for sdxl currently